scripts/mutect2_wdl/mutect2_template.json

```diff
@@ -18,7 +18,7 @@
   "Mutect2.is_run_orientation_bias_filter": "$__is_run_orientation_bias_filter__",
   "Mutect2.is_run_oncotator": "$__is_run_oncotator__",
   "Mutect2.gatk_docker": "broadinstitute/gatk:4.beta.5",
-  "Mutect2.oncotator_docker": "broadinstitute/oncotator:1.9.3.0",
+  "Mutect2.oncotator_docker": "broadinstitute/oncotator:1.9.6.1",
```

Comment:
Thanks!

Reply:
The default Oncotator Docker in the CNV WDL is broadinstitute/oncotator:1.9.5.0-eval-gatk-protected, do we need to update?  I saw some chatter in the DSDE Comms Slack channel about the CNV pair workflow failing on FireCloud at the Oncotator step, not sure if this has anything to do with that.  @bshifaw

Reply:
CNV docker is different than the M2 docker.  Each is correct in master.  

@bshifaw Make sure you have the latest WDL from `master`.  I made an update to fix an issue with the CNV docker.  I have confirmed that the fix works on ~12 samples.